mkt/developers/models.py

```diff
@@ -193,41 +191,6 @@ class Meta:
         db_table = 'user_inapp_keys'
 
 
-class PreloadTestPlan(ModelBase):
-    addon = models.ForeignKey('webapps.Webapp')
```

Comment:
maybe i'll just wipe the table and trust ops can pull all the PDFs that are sitting in a folder somewhere on some server